tests/controls/radiobuttontest.cpp

```diff
@@ -150,4 +152,41 @@ void RadioButtonTestCase::Group()
     wxDELETE(g2radio1);
 }
 
+void RadioButtonTestCase::Single()
+{
+    //Create a group of 2 buttons, having second button selected
+    wxRadioButton* gradio0 = new wxRadioButton(wxTheApp->GetTopWindow(),
+        wxID_ANY, "wxRadioButton",
+        wxDefaultPosition,
+        wxDefaultSize, wxRB_GROUP);
+
+    wxRadioButton* gradio1 = new wxRadioButton(wxTheApp->GetTopWindow(),
+        wxID_ANY, "wxRadioButton");
+
+    gradio1->SetValue(true);
+
+    //Create a "single" button (by default it will not be selected)
+    wxRadioButton* sradio = new wxRadioButton(wxTheApp->GetTopWindow(),
+        wxID_ANY, "wxRadioButton",
+        wxDefaultPosition,
+        wxDefaultSize, wxRB_SINGLE);
+
+    //Create a non-grouped button and select it
+    wxRadioButton* ngradio = new wxRadioButton(wxTheApp->GetTopWindow(),
+        wxID_ANY, "wxRadioButton");
+
+    ngradio->SetValue(true);
+
+    //Select the "single" button
+    sradio->SetValue(true);
+
+    CPPUNIT_ASSERT(gradio1->GetValue());
+    CPPUNIT_ASSERT(ngradio->GetValue());
+
+    wxDELETE(gradio0);
```

Comment:
I'd rather use `wxScopedPtr` than manual deletes (even though this should be safe for now if `CHECK()` is used, it's always a danger that it could be easily broken later), but if we don't, let's use just plain `delete` here, what's the point of using a macro when we don't have to?